src/Particle.js

```diff
@@ -72,6 +72,24 @@ class Particle {
 		}, method: 'post', context });
 	}
 
+	/**
+	 * If login failed with an 'mfa_required' error, this must be called with a valid OTP code to login
+	 * @param  {Object} options Options for this API call
+	 * @param  {String} options.mfaToken Given as 'mfa_token' in the error body of `.login()`.
+	 * @param  {String} options.otp      Current one-time-password generated from the authentication application
+	 * @param  {Number} options.context  Request context
+	 * @return {Promise}
+	 */
+	loginMfa({ mfaToken: mfa_token, otp, context }) {
```

Comment:
Let's call this `sendOtp` since it is what it does

Reply:
Sounds good.